Installer.cls

```diff
@@ -42,12 +49,15 @@ XData PM [ XMLNamespace = INSTALLER ]
 /// This is a method generator whose code is generated by XGL.
 ClassMethod setup(ByRef pVars, pLogLevel As %Integer, pInstaller As %Installer.Installer, pLogger As %Installer.AbstractLogger) As %Status [ CodeMode = objectgenerator, Internal ]
 {
-  #; Let our XGL document generate code for this method. 
+  // Let our XGL document generate code for this method. 
   Do %code.WriteLine(" If $Data(pVars, path)#10,path'="""" Set pVars(""MODDIR"")=path")
   Do %code.WriteLine(" Set pVars(""CURNS"")=$namespace")
   Do %code.WriteLine(" Set prevNS = $namespace")
   Do %code.WriteLine(" New $Namespace")
   Do %code.WriteLine(" Set $Namespace = ""%SYS"" ")
+  // Check whether or not corelib exists
+  Do %code.WriteLine(" Set tLibExists = ##class(SYS.Database).%ExistsId(""CORELIB"")")
+  Do %code.WriteLine(" If 'tLibExists { Set tSC = $$$ERROR($$$GeneralError,""CORELIB database does not exist, cannot install IPM!"") Quit tSC}")
```

Comment:
Make a comment that we should never reach here because the installer manifest should create corelib if it doesn't exist